components/motor/gpio/basic_test.go

```diff
@@ -22,18 +23,20 @@ func TestMotorABPWM(t *testing.T) {
 	b := &fakeboard.Board{GPIOPins: map[string]*fakeboard.GPIOPin{}}
 	logger := golog.NewTestLogger(t)
 
+	mc := config.Component{}
```

Comment:
Give this an actual name for future tests. Also, would you like to write a test to check that an error is being returned?